app/js/createUserObject.ts

```diff
@@ -77,38 +94,52 @@ function secondsToMinutes(time: number) {
 function findTestName(tests: Array<Test>, testId: number) {
     let testName = "None Assigned"
     tests.forEach(function(test) {
-        if (testId == test.id) {
+        if (testId === test.id) {
             testName = test.name
         }
-    })
+    });
     return testName
-}
+};
+
+function findCategoryName(categories: Array<Categories>, categoriesId: number) {
```

Comment:
surely the second parameter should be `categoryId` as it it the ID of a single category